### DISCUSSION TITLE
Cannot Apply Student Developer Pack Because of Unintelligible Request

### DISCUSSION THREAD

**Discussion by MidCoard** [community/community#52304] [GitHub Education]
When I apply for the student developer pack, the notice that "You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio." is shown. I have completed my profile with my full name and school email address, but it still shows this notice. What should I do? I think there should be a clearer description.

By the way, why should I make my full name public? I don't think it's a good idea. This is my privacy also my email address. I don't think there is a need to make it public. Verifying my identity with official documents that I have provided and the school email-address is enough.

Also, I can use National Valid Student Status Certificate to verify my identity. This is an official link to the certificate but there is no place to fill. In the second page of the application, there is only a place to upload a photo or take a photo.

One more question, I have successfully applied for the student developer pack before. Is there a way to renewal it instead of applying it again? I think there should be a way.

**Comment by MidCoard** [comment]
Why github uses my camera when I click the Process My Application button in the second page of application. There is no notice and I think this is unreasonable as there is no need to get my personal information when I click the button!
![ev](https://user-images.githubusercontent.com/24197013/230706852-426ce0cf-6f28-4445-badc-ae9b538417ef.jpg)

**Comment by samus-aran** [comment]
Hi @MidCoard 👋 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Let us know how you get on 👍

**Reply by MidCoard** [reply]
I have provided a document that is a UESTC student card with a handwritten English translation on a paper.

I have changed my Github profile name to the full name showing in the document that I have provided and the Github profile email to my school email.

I have logged out and logged back into Github.

But I still got "You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.".

**Reply by MidCoard** [reply]
Is there a necessary to change my username to real name?

**Reply by samus-aran** [reply]
Hi @MidCoard 👋 

Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either so ensure you take a photo as straight as possible. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps.

We use your username, profile name and email address and other parts. If you've filled in one of those areas with details that correspond to your evidence it should be picked up.

If you are satisfied you have entered the correct information on your profile then I would ask you to re-look at your image/photo and assess if it is readable. 👍

**Reply by MidCoard** [reply]
I take pictures with my mobile phone, can the picture be horizontal?

By the way, if I apply for it successfully, can I change my profile name to my nickname?
